feature-libs/product-configurator/common/shared/utils/common-configurator-utils.service.ts

```diff
@@ -65,18 +68,23 @@ export class CommonConfiguratorUtilsService {
    * @param cart Cart
    * @returns Cart identifier
    */
-  public getCartId(cart: Cart): string {
+  getCartId(cart: Cart): string {
     return cart.user.uid === OCC_USER_ID_ANONYMOUS ? cart.guid : cart.code;
   }
+
   /**
-   * Gets cart user
+   * Gets user assigned to a cart which we can use for OCC requests
    * @param cart Cart
-   * @returns User identifier
+   * @returns Observable of user identifiers
```

Comment:
Is it correct that the observable contains user identifiers not just one identifier?